examples/fabric-gateway/test.sh

```diff
@@ -0,0 +1,89 @@
+#--------create FG
```

Comment:
A file named `test.sh` is unconventional. This may be better as a README.md or a TESTING.md that breaks this document up into explanations and headings that perform the steps outlined below.

What we really need is an E2E testing credential for Fabric resources in the Terraform provider.

Reply:
Changed this to a markdown file and formatted accordingly.

Reply:
The formatting could be improved.

* the code blocks could be captured in blocks like this:
   ~~~
   ```sh
   terraform something
   ```
   ~~~
* the filename could be TEST.md
* the headings could be in better descending order: `#` (only at the top), `##` (for sub sections), `###` (for sub-sub sections)
* newlines could be added between the headings, sub-headings, and code-blocks

We should add markdown linters for these concerns though, I won't fault you for it. We can chase these improvements in future PRs.

Reply:
Ack. Thanks for pointing out. This was supposed to be an executable shell script for internal testing. I missed some formatting due to last minute changes.  I have now updated this file with the correct formatting.